operator/handler/schedule_test.go

```diff
@@ -184,6 +184,33 @@ func TestScheduleHandler_mergeBackendWithDefaults(t *testing.T) {
 	}
 }
 
+func TestScheduleHandler_mergePodSecurityContextWithDefaults(t *testing.T) {
+	var nilPodSecurityContextPointer *corev1.PodSecurityContext
```

Comment:
why having `nil` as a variable?
```suggestion
```

Reply:
Isn't this the value which triggers the panic error?

Reply:
See my suggestion "above", just use `nil`, since `givenSchedulePodSecurityContext` is a pointer type `*corev1.PodSecurityContext`